cmds/fmap/fmap.go

```diff
@@ -64,11 +77,24 @@ func printFMap(f *fmap.FMap, m *fmap.FMapMetadata) {
 	}
 }
 
+func btoi(b bool) int {
+	if b {
+		return 1
+	}
+	return 0
+}
+
 func main() {
 	flag.Parse()
 
-	if *summary && *read >= 0 {
-		log.Fatal("Both flags cannot be used at once")
+	// Validate flags
+	if btoi(*summary)+btoi(*read >= 0)+btoi(*checksum != "") > 1 {
```

Comment:
I've been thinking of a thing like this but ... I am not real comfortable with this. 

The problem is that we can get conflicting flags, and I keep thinking there has to be a way to solve it. Maybe we should look at how the flags package works a little more. There is a default interface and it sets up a global thing but we maybe we should be using it differently.

In any event why not
var btoi = map[bool] int{false: 0, true:1, }?